am2320/am2320.go

```diff
@@ -0,0 +1,181 @@
+// Copyright 2024 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+//
```

Comment:
Make this line empty otherwise the comment will be shown in the go documentation.